src/main/java/com/booleanuk/extension/Extension.java

```diff
@@ -5,7 +5,14 @@ public class Extension {
 
     public void stepOne() {
         // TODO: 1. Write a for loop that adds all the even numbers between 0 and 6 (0, 2, 4, 6) to evenNums
-
+        numsEven = new int[4];
+        int counter = 0;
+        for (int i = 0; i <= 6; ++i) {
+            if (i % 2 == 0) {
+                numsEven[counter] = i;
+                counter++;
+            }
+        }
```

Comment:
This is good. If you want an extra challenge, can you think of a way to restructure your code so that we don't need the counter variable?